desktop/cmp/treemap/TreeMap.js

```diff
@@ -302,6 +302,8 @@ class LocalModel {
         this.chart.series[0].data.forEach(node => {
             node.select(toSelect.has(node.id), true);
         });
+
+        gridModel.ensureSelectionVisible();
     }
```

Comment:
This should be made null-safe, as it is on 292.  The associated GridModel is optional in TreeMapModel.

Reply:
Done.